goalgorilla/open_social
# Issue #FOR-333 by rolki: Update social_gdpr accordingly with the data_policy

## Problem
The data_policy module was updated [here](https://github.com/goalgorilla/data_policy/pull/22) and as a result, our social_gdpr module doesn't work with multiple entities and a new version of that module

## Solution
* This module should work with multiple entities  *

## Issue tracker
https://getopensocial.atlassian.net/browse/FOR-333 

## How to test
*For example*
- [] Go to the `/data-policy/revisions` page and you should see the list of revisions for active entities
- [] Go to the `/data-policy` page and you should see only active revisions accordingly with "Consent text" on the `/admin/config/people/data-policy/settings` page

## Screenshots
![image](https://user-images.githubusercontent.com/50984627/87758990-38713380-c816-11ea-8e58-d1542a43678a.png)
![image](https://user-images.githubusercontent.com/50984627/87759003-3effab00-c816-11ea-8546-d361b1526cda.png)
![image](https://user-images.githubusercontent.com/50984627/87759030-48891300-c816-11ea-9d64-c501939a9a40.png)
![image](https://user-images.githubusercontent.com/50984627/87759065-58085c00-c816-11ea-9e68-16c935857ca0.png)

## Release notes
**These changes should be merged after https://github.com/goalgorilla/data_policy/pull/22, because this version of the module will not be work with the old version of data policy.**
